### PR TITLE
[G2M] migrate to bolt.js

### PR DESCRIPTION
**Pull request by woozyking**
- much simpler workflow, including the previously tedious interactive views, and no need to maintain how requests are verified (signing secret verification)
- removed severely out-of-date commands
- retained underutilized, but redeemable command logics to `modules/need-revision`
- changed dep management CLI from yarn to npm (v8+)
- all commands can stick to a single entry URL at https://o1u4dgxs7a.execute-api.us-east-1.amazonaws.com/dev/slack/events
- configure with `serverless-offline` for proper local testing (without the explicit `DEPLOYED` determination)

known issue:
- ~`/demo` view handler isn't working properly atm, but the invocation flow is much better now (per note ^)~ demo actually "works" now, but doesn't work to create calendar event because of the outdated google API/token
- obviously the ^ mentioned `need-revision` commands need a follow-up to get things working

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,227 +1,47 @@
...
+    text: 'snoke',
+    api_app_id: 'A6HPM5VC0',
+    is_enterprise_install: 'false',
+    response_url: 'https://hooks.slack.com/commands/T1FDR4NG3/2956432472786/J8Q17JPFukiLurI8zbVJzTCX',
+    trigger_id: '2956462218819.49467158547.0dae80ec6d595b045efcf7a8fee079e7'
   }
+*/
+Object.entries(commands).forEach(([name, { listener }]) => {
+  app.command(`/${name}`, listener)
 })
```
Comment (vxsl) [inline app.js:34]: :sunglasses:

File: app.js [hunk 2]
```diff
@@ -1,227 +1,47 @@
...
-  // app log
-  // eslint-disable-next-line no-console
-  console.log(`[${logLevel}] - ${statusCode} - ${message}`)
-  if (logLevel === 'ERROR') {
-    console.error(`[ERROR] ${message}`, err.stack || err)
+// interactive views, by callback_id
+// TODO: ones that require rework are commented out
+Object.entries(commands).forEach(([name, { viewHandler }]) => {
+  if (viewHandler) {
+    app.view(name,  viewHandler)
```
Comment (vxsl) [inline app.js:40]: any reason not to combine the `forEach`s?
```js
Object.entries(commands).forEach(([name, { listener, viewHandler }]) => {
  app.command(`/${name}`, listener)
  if (viewHandler) {
    app.view(name,  viewHandler)
  }
}  
```

File: modules/commands/food.js [hunk 3]
```diff
@@ -87,38 +86,20 @@ const worker = async ({ response_url, ...params }) => {
...
-        return res.status(200).json({ response_type: 'ephemeral', text: 'Failed to search' })
-      }
-      return res.status(200).json({ response_type: 'ephemeral', text: 'Searching for food...' })
-    })
-  } else {
-    worker(payload).catch(console.error)
-    return res.status(200).json({ response_type: 'ephemeral', text: 'Searching for food...' })
-  }
+  await invokeSlackWorker({ type: 'food', payload })
+  await respond('Searching for food...')
```
Comment (vxsl) [inline modules/commands/food.js:102]: deprecated by covid :sob:

**Review by vxsl — APPROVED** [review]
lgtm!

**Review by hyx131 — APPROVED** [review]
much cleaner 🤩

File: app.js (hunk 1 above)
Comment (woozyking) [reply app.js:34]: truly dynamic

File: app.js (hunk 2 above)
Comment (woozyking) [reply app.js:40]: probably an afterthought. no good reason and they should be combined

File: modules/commands/food.js (hunk 3 above)
Comment (woozyking) [reply modules/commands/food.js:102]: lol if we can capture the sender's geo, then this can be useful again

**Pull request merged**